source/js/ellipsis-util.js

```diff
@@ -0,0 +1,99 @@
+var ellipsis = require("html-ellipsis");
+var characterPerLine = require("./characters-per-line");
+
+var EllipsisUtil = {
+  /***
+   * @param {Object} lineStyles: Node with DOM elements that has a string
+   * @return {Boolean}
+   */
+  hasSpecificHeight: function(lineStyles) {
+    return lineStyles.height != "auto" && parseInt(lineStyles.height) != 0;
```

Comment:
comparison of "lineStyles.height != "auto"" may cause unexpected  type coercionm, we should replace it with "!==" comparison. Same for the parseInt() !=0 comparision

Reply:
done